src/composables/useUniqueId.ts

```diff
@@ -0,0 +1,3 @@
+import { nanoid } from 'nanoid'
+
+export default nanoid as () => string
```

Comment:
nit: I think you're missing an editor setting or something; I've noticed when you create new files there is no newline at the end (causing this alert)

Reply:
Will check. For now adding the trailing line manually